Cargo.toml

```diff
@@ -7,7 +7,7 @@ members = ["cli", "lib", "lib/gen-protos", "lib/proc-macros", "lib/testutils"]
 [workspace.package]
 version = "0.22.0"
 license = "Apache-2.0"
-rust-version = "1.76" # NOTE: remember to update CI, contributing.md, changelog.md, install-and-setup.md, and flake.nix
```

Comment:
flake.nix seems to use nightly Rust since 749481e5527c092da9096504e08c6aa065f598bb